hoot-core/src/main/cpp/hoot/core/util/Settings.cpp

```diff
@@ -120,9 +120,7 @@ class JsonLoader
         continue;
       if (!_s->hasKey(name))
       {
-        LOG_WARN("Unknown JSON setting: (" << name << ")");
-        //  Don't allow unknown settings to be loaded
-        continue;
+        throw HootException("Unknown JSON setting: (" + name + ")");
```

Comment:
That's strange.  Yeah, that was part of different PR you reviewed early.  I think I inadvertently made the same change on this branch...probably b/c it was helping with the debugging.  I'm not understanding why git sees a diff there...code here looks exactly the same as develop branch.  Maybe just merge it and I'll keep an eye on it after the merge.  I think if I take that change out and commit it, then git will overwrite the whole change, since that commit will be later than the one in develop.